### PR TITLE
feature: added button

### PR DESCRIPTION
**Pull request by omgaikwad**
Added button:
-Solid Primary Button
-Outline Button
-Link Button
-Icon Button
-Floating Action Button

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.btn {
+  box-sizing: border-box;
+  border: none;
+  padding: 0.6rem 1rem;
```
Comment (Varun-28) [inline components/button/button.css:4]: for size less than 1rem you can use px, rest all is fine 👍

File: components/button/button.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="shortcut icon"
+      href="../../assets/favicon/favicon-red-black.ico"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="../../css/main.css" />
```
Comment (Varun-28) [inline components/button/button.html:22]: add all the css files in one css file as import and import that file in your component css file, although you can do it at last for all components, as you wish.

**Review by shivampandey0 — APPROVED** [review]
LGTM

File: components/button/button.css (hunk 1 above)
Comment (omgaikwad) [reply components/button/button.css:4]: Sure

File: components/button/button.html (hunk 2 above)
Comment (omgaikwad) [reply components/button/button.html:22]: Yeah I will do in the end

**Pull request merged**